bears/c_languages/GNUIndentBear.py

```diff
@@ -23,18 +23,169 @@ class GNUIndentBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=80,
                          use_spaces: bool=True,
+                         blank_lines_after_declarations: bool=False,
```

Comment:
not really

2016-06-17 14:51 GMT+01:00 Abhay Raizada <notifications@github.com>:

> In bears/c_languages/GNUIndentBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/519#discussion_r67512986>
> :
>
> > @@ -23,18 +23,169 @@ class GNUIndentBear:
> >      def create_arguments(filename, file, config_file,
> >                           max_line_length: int=80,
> >                           use_spaces: bool=True,
> > +                         blank_lines_after_declarations: bool=False,
>
> do these follow any order?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/519/files/4e617267213ef6a8fbac778c54e69f17da5bf6d9#r67512986>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xorZdJG7IzNbI7Y5Auy9ImG1Vbndzks5qMqZWgaJpZM4I02l0>
> .
>